### DISCUSSION TITLE
Building a vdproj setup project

### DISCUSSION THREAD

**Discussion by hadastoroP** [actions/virtual-environments#7238] [General questions]
Hi,
I am using the 2019 docker image to build my .Net Framework solution.
My solution includes a Project Setup project (vdproj extension)  to create an Installer for my application.
I can run the installer from command line on my own machine using the devenv.com (since msbuild does not build setup projects)
I have read in the following article :
https://spin.atomicobject.com/2022/05/17/visual-studio-installer-azure/
that devenv.com should br available in this image.... but I cannot find it on the docker container...

Can anyone please help?

**Comment by acnicholls** [comment]
for anyone looking for help on this, use this action to setup the PATH variable for `devenv.com`  

seanmiddleditch/gha-setup-vsdevenv@v4

then just use `devenv` in your CLI commands in actions
